services/brig/src/Brig/API/Error.hs

```diff
@@ -484,3 +486,13 @@ federationNotImplemented qualified =
     rendered = LT.intercalate ", " . toList . fmap (LT.fromStrict . renderMapping) $ qualified
     renderMapping IdMapping {idMappingLocal, idMappingGlobal} =
       idToText idMappingLocal <> " -> " <> renderQualifiedId idMappingGlobal
+
+-- (the tautological constraint in the type signature is added so that once we remove the
+-- feature, ghc will guide us here.)
+customerExtensionBlockedDomain :: (DomainsBlockedForRegistration ~ DomainsBlockedForRegistration) => Domain -> Wai.Error
```

Comment:
I miss agda now.

What you want here is a witness that ` (x : Domain) -> (x  <= domainsBlocked)`

But this is a fine workaround :)

Reply:
thank you.  :)